### PR TITLE
update br 0211

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md [hunk 1]
```diff
@@ -14,66 +14,103 @@
 
 - BR 编译完成。如何编译 BR，请参见[编译 BR](2.compile-br.md)。
 
+- 已下载 [nebula-agent](https://github.com/vesoft-inc/nebula-agent) 并在集群中的每个主机上运行代理服务。
```
Comment (pengweisong) [inline docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md:17]: could add bold on "每个"

File: docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md [hunk 2]
```diff
@@ -14,66 +14,103 @@
...
+
 - 确认没有应用程序连接到待恢复数据的 Nebula Graph 集群。
 
 - 确认集群的拓扑结构一致，即原集群和目标集群的主机数量一致。
```
Comment (pengweisong) [inline docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md:21]: 主机数量一致--> 主机数量一致，且每个主机数据文件夹数量分布一致。

File: docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md [hunk 3]
```diff
@@ -14,66 +14,103 @@
...
-   例如，可以使用以下命令，将本地 `/home/foesa/backup` 路径中的数据恢复到地址为`192.168.*.*:9559`，meta/storage 服务所在主机的用户名为`root`的集群：
-
-   ```
-   [root]# ./bin/br restore full --meta "192.168.*.*:9559" --storage "local:///home/test/backup" --name "BACKUP_2021_07_29_07_55_08" --user "root"
-   ```
+  ```
+  $ ./bin/br restore full --meta <ip_address> --storage <storage_path> --name <backup_name>
+  ```
+
+  例如，可以使用以下命令，将本地 `/home/foesa/backup` 路径中的数据恢复到地址为`192.168.*.*:9559`，meta/storage 服务所在主机的用户名为`root`的集群：
```
Comment (pengweisong) [inline docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md:64]: the address should be modified, and the user should be removed

File: docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md [hunk 4]
```diff
@@ -14,66 +14,103 @@
...
+  | --storage | string | 是 | 无 | BR 备份数据存储位置，格式为：\<Schema\>://\<PATH\> <br>Schema：可选值为 `local` 和 `s3`。选择 s3 时，需要填写`s3.access_key`、`s3.endpoint`、`s3.region`和 `s3.secret_key`。<br>PATH：存储位置的路径。|
+  | --s3.access_key | string | 否 | 无 | 用于标识用户。 |
+  | --s3.endpoint | string | 否 | 无 | S3 对外服务的访问域名的 URL，指定 http 或 https。 |
+  | --s3.region | string | 否 | 无 | 数据中心所在物理位置。 |
+  | --s3.secret_key| string | 否 | 无 | 用户用于加密签名字符串和用来验证签名字符串的密钥，必须保密。 |
+
+3. 如果在备份期间发现任何错误，用户可以使用以下命令清理临时文件。该命令将清理集群和外部存储中的文件，同时用户也可以使用该命令清理外部存储中的旧的备份文件。
+
+  ```
+  $ ./bin/br clearnup --meta <ip_address> --storage <storage_path> --name <backup_name>
```
Comment (pengweisong) [inline docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md:101]: cleanup spelling.

**Pull request merged**